sdk/ml/azure-ai-ml/azure/ai/ml/entities/_schedule/schedule.py

```diff
@@ -66,7 +64,6 @@ def __init__(
         super().__init__(name=name, description=description, tags=tags, properties=properties, **kwargs)
         self.trigger = trigger
         self.display_name = display_name
-        self.create_job = create_job
```

Comment:
not having `create_job` in the `Schedule` contract is a breaking change and releasing this code will be blocked by the Azure SDK team. Please sync with me offline so I can give you the full context